src/lib/components/Molecule3dViewer.js

```diff
@@ -118,6 +128,48 @@ Molecule3dViewer.propTypes = {
         negativeVolumetricColor: PropTypes.string,
     }),
 
+    /**
+     * Zoom the current view by a constant factor, with optional parameters
+     * to modify the duration and motion of the zoom animation.
+     */
+    zoom: PropTypes.exact({
+        /**
+         * Magnification factor. Values greater than 1 will zoom,
+         * in, less than one will zoom out. Default 2.
+         */
+        factor: PropTypes.number,
+        /**
+         * An optional parameter that denotes the duration of a
+         * zoom animation.
```

Comment:
```suggestion
         * zoom animation, in milliseconds.
```
Never stated in their docs but must be ms? Same for `zoomTo`.

Reply:
:+1:  Fixed in fe76521.